doc/syntax/comments.rdoc

```diff
@@ -35,3 +35,188 @@ syntax error:
     Will not work
     =end
   end
+
+= Magic Comments
+
+While comments are typically ignored by Ruby, special "magic comments" contain
+directives that affect how the code is interpreted.
+
+Top-level magic comments must start on the first line, or on the second line if
+the first line looks like `#! shebang line`.
+
+NOTE: Magic comments affect only the file in which they appear;
+other files are unaffected.
+
+== Alternative syntax
+
+Magic comment may consist of a single directive multiple (as in the examples above).
```

Comment:
"single directive multiple"?.. Also, there seems to be no example above :thinking:

Reply:
That's what happens when I'm too tired (and I move a section too fast) 😅